src/twisted/conch/test/test_forwarding.py

```diff
@@ -67,15 +42,11 @@ def test_channelOpenHostnameRequests(self):
         sut = forwarding.SSHConnectForwardingChannel(
             hostport=('fwd.example.org', 1234))
         # Patch channel and resolver to not touch the network.
-        sut._reactor = MemoryReactorClock()
-        self.patchHostnameEndpointResolver(
-            request=('fwd.example.org', 1234),
-            response=(AF_INET6 ,'::1'),
-            )
-
+        memoryReactor = MemoryReactorClock()
+        sut._reactor = deterministicResolvingReactor(memoryReactor, ['::1'])
```

Comment:
I didn't realize I'd actually pooched the assertion, but I did realize this was sub-optimal.  See https://github.com/twisted/twisted/commit/f19151e59bb9b64306ce023d85f5621cdb4ad473 - once that's landed, I will adjust some of these tests.